components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/model/User.java

```diff
@@ -18,13 +18,17 @@
 
 package org.wso2.carbon.identity.action.execution.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * This class models the User.
  * User is the entity that represents the user for whom the action is triggered for.
  */
 public class User {
 
     private String id;
+    private final List<UserClaim> userClaims =  new ArrayList<>();
```

Comment:
This will add the user claims to the request as 'userClaims' right ?
Our proposal includes 'claims' as it if of user